media-gfx/blender/blender-9999.ebuild

```diff
@@ -33,12 +33,13 @@ fi
 
 LICENSE="|| ( GPL-3 BL )"
 SLOT="${PV%.*}"
-IUSE="+bullet +fluid +openexr +tbb
-	alembic collada +color-management cuda +cycles cycles-bin-kernels
-	debug doc +embree +ffmpeg +fftw +gmp gnome hip jack jemalloc jpeg2k
-	man +nanovdb ndof nls openal +oidn +openmp +openpgl +opensubdiv
-	+openvdb optix osl +pdf +potrace +pugixml pulseaudio sdl
-	+sndfile test +tiff valgrind wayland +webp X"
+IUSE="
+	alembic +bullet collada +color-management cuda +cycles +cycles-bin-kernels
+	debug doc +embree experimental +ffmpeg +fftw +fluid +gmp gnome hip jack
+	jemalloc jpeg2k man +nanovdb ndof nls +oidn oneapi openal +openexr +openmp openpgl
```

Comment:
`openpgl` is not on per default any more, I'm guessing this is a mistake from reordering the useflags?